.github/workflows/push_tests.yml

```diff
@@ -140,7 +140,15 @@ jobs:
         # https://pytorch.org/docs/stable/notes/randomness.html#avoiding-nondeterministic-algorithms
         CUBLAS_WORKSPACE_CONFIG: :16:8
       run: |
-        python -m pytest -n 1 --max-worker-restart=0 --dist=loadfile \
+        # DeprecatedAttentionBlockTests::test_conversion_when_using_device_map fails 
```

Comment:
do you know why this test would fail with `-n 1 --dist=loadfile` option? also did you tested it anywhere that it would work without this option?

I tried both 
```bash
pytest -n 1 --dist=loadfile tests/models/test_attention_processor.py::DeprecatedAttentionBlockTests::test_conversion_when_using_device_map
```
and 
```bash
pytest tests/models/test_attention_processor.py::DeprecatedAttentionBlockTests::test_conversion_when_using_device_map
```

both were passing on my machine

Reply:
Yeah same. I tested on DGX and audace and both passed. So, no idea. Didn’t dig deeper because the blocks are deprecated anyway.

Reply:
so the test would pass if we remove this line "-n 1 --dist=loadfile" ?

Reply:
On our CI, yes.

Reply:
ok! I'm ok with the change then 
what do you think @DN6 ?